### PR TITLE
bug 1186899 - improve browserstack intern config

### PR DESCRIPTION
**Pull request by groovecoder**
Though I backed off from running Intern + BrowserStack during chief deploy, I found these improvements to make to our browserstack configuration.

### CODE REVIEW

File: tests/ui/intern-browserstack.js [hunk 1]
```diff
@@ -3,14 +3,26 @@
 // packages, suites, excludeInstrumentation, and (if you want functional tests) functionalSuites.
 define(['./_base', './_cli', 'intern'], function(config, cli, intern) {
 
+    // Name of the tunnel class to use for WebDriver tests
+    config.tunnel = 'BrowserStackTunnel';
+
+    config.capabilities = {
+        'browserstack.ie.noFlash': true,
+        'browserstack.ie.enablePopups': true
+    }
```
Comment (darkwing) [inline tests/ui/intern-browserstack.js]: nit: semicolon

File: tests/ui/intern-browserstack.js [hunk 2]
```diff
@@ -3,14 +3,26 @@
...
+
     // Browsers to run integration testing against.
     config.environments = [
-        { browserName: 'firefox', version: ['40'], platform: [ 'MAC' ] },
-        { browserName: 'chrome', version: ['44'], platform: [ 'MAC' ] },
+        { browserName: 'firefox', platform: [ 'MAC', 'WIN8' ] },
```
Comment (darkwing) [inline tests/ui/intern-browserstack.js]: nit: no spaces at beginning and end of array

File: tests/ui/intern-browserstack.js [hunk 3]
```diff
@@ -3,14 +3,26 @@
...
+        { browserName: 'chrome', platform: [ 'MAC', 'WIN8' ] },
```
Comment (darkwing) [inline tests/ui/intern-browserstack.js]: ^

**Comment by groovecoder** [comment]
Updated. Not sure why it can't auto-merge. Let me hit some other PRs and try to rebase this.

**Comment by groovecoder** [comment]
Grr ... still unable to merge this. :(

**Pull request closed without merging**